src/modlist.cpp

```diff
@@ -1074,14 +1047,14 @@ int ModList::dropPriority(int row, const QModelIndex& parent) const
 
   int newPriority = 0;
   {
-    if ((row < 0) || (row > static_cast<int>(m_Profile->numRegularMods()))) {
-      newPriority = m_Profile->numRegularMods() + 1;
+    if (row < 0 || row >= rowCount()) {
+      newPriority = std::numeric_limits<int>::max();
```

Comment:
One thing that's getting pretty confusing is the usage of `INT_MIN`, `UINT_MAX`, `std::numeric_limits<int>::max()`, etc. This line I think means "at the end"? I know we're already using these constants all over the place, but I hate it.

Reply:
I removed usage of `INT_MIN` to mean "not a fixed priority" because I agree it was misleading. Here the value means "the biggest you can", not really "at the end" (well depends what you call "at the end"), and it let the profile fix it.